common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -28,6 +28,9 @@ class @Problem
     problem_prefix = @element_id.replace(/problem_/,'')
     @inputs = @$("[id^='input_#{problem_prefix}_']")
     @$('div.action button').click @refreshAnswers
+    @questionTitle = @$('h3.problem-header')
```

Comment:
We prefer to rely just on class names and not include the HTML tag types. So just .problem-header, .review-btn, etc.